docs/it/week03/03.md

```diff
@@ -0,0 +1,27 @@
+---
+lang-ref: ch.03
+titolo: Week 3
```

Comment:
This is going to break our website.
There's a program reading these headers.
You cannot change the name of the variables! LOL